src/Draco.Compiler/Internal/Symbols/FunctionSymbol.cs

```diff
@@ -75,6 +75,13 @@ public delegate IOperand CodegenDelegate(
         _ => throw new System.ArgumentOutOfRangeException(nameof(token)),
     };
 
+    /// <summary>
+    /// The receiver of this function, if it has one.
+    /// </summary>
+    public TypeSymbol? Receiver => this.IsStatic
```

Comment:
Maybe rename to `ReceiverType`, I'm not sure. Would we expect a parameter called `Receiver` for example?